### DISCUSSION TITLE
[Scam Alert] United Solar یونایتد سولار

### DISCUSSION THREAD

**Discussion by sadegh98** [Webamoozcom/warning-list#170] [Investigation بررسی]
سلام به همه
چند روزی هست که توسط یکی از آشناها با یک پروژه پانزی آشنا شدم. خواستم اینجا مطرحش کنم که شما عزیزان هم به بقیه اطلاع رسانی کنید که گرفتار و اسیر نشن و سرمایشون رو از دست ندن.

شرح کلاهبرداری :
یک شرکت استرالیایی که در زمینه تولید و نگهداری از پنل های خورشیدی کار میکنه ، میاد روزانه و ماهانه پنل ها رو اجاره میده. افراد با خیر یکسری بسته ها ( به صورت ارز تتر ) در پنل های خورشیدی شریک میشن و بعد از مدتی هم سود فضاییشون رو دریافت میکنن.
همچنین شما با زیرمجموعه گیری میتونید درآمدتون رو بیشتر کنید.

میزان سود افراد از سرمایه گذاری :
![image](https://user-images.githubusercontent.com/62065448/209579404-8e8164e9-9e63-498f-8f82-376cc95e1c47.png)
میزان سود افراد از زیرمجموعه گیری :

![image](https://user-images.githubusercontent.com/62065448/209579464-e50ccb55-e689-4c4c-8a15-d21a3e513987.png)

وب سایتشون :

https://www.unitedsolarinfinity.com/
archive :
https://web.archive.org/web/20221226195208/https://www.unitedsolarinfinity.com/

لینک اپلیکیشن اندروید :
https://play.google.com/store/apps/details?id=com.yesapp.unitedsolar2
archive :
https://web.archive.org/web/20221226195450/https://play.google.com/store/apps/details?id=com.yesapp.unitedsolar2

لینک کانال تلگرام که تمام مراحل رو مو به مو توضیح داده :

https://t.me/mteterms
archive:
https://web.archive.org/web/20221226195712/https://t.me/s/mteterms

**Comment by Andy0619** [comment]
你好，你的来信已收到

**Comment by micro73** [comment]
1. هویز سایت پنهان شده است – دامنه سایت در تاریخ 18/09/2022 به ثبت رسیده است.
2. ثبت نام و ورود به سایت از دامنه دیگری انجام ‌می‌شود.
3. ثبت نام حتما با کد دعوت و از ادرس https://client.unitedsolarinfinity.com/ باید صورت بگیرید.
4. صفحه ورود و ثبت نام مخصوص موبایل طراحی شده است که اگر با سیستم وارد این صفحه شوید، کاملا مشخص است.
5. موقع ثبت نام درخواست شماره تلفن دارد که کد ایران به صورت پیش فرض در ابتدای لیست قرارداده شده است که نشان می‌دهد تمرکز روی مردم ایران است.
6. هویت گردانندگان مشخص نیست.
7. مشخص نیست این پنل‌های خورشیدی کجا واقع شده است.

**Reply by Rezamuhammadi** [reply]
برای مورد 5 شما فکر میکردم لیستو از روی آی پی یا تایم زون میخونه که اینطور نبود با هر آی پی یا تایمی ایران بالاس و جالب تر اینکه فارسی نوشته. اینا دیگه خیلی تابلو بین المللی ان

**Comment by libertyir** [comment]
اطلاعات تکمیلی اینکه 
پروژه کلا ایرانیه و رو سرورهای ایران مستقره
https://194.59.170.243/
الان هم سطح 6 باینری هستند 
https://client.unitedsolarinfinity.com
api: https://api.unitedsolar-vip.com/api/
https://47.242.5.217/
https://47.242.224.66/
https://47.242.3.142/
https://8.210.131.140/login?redirect=/dashboard


![image](https://user-images.githubusercontent.com/90643392/233174097-f0a1b40a-815a-4a0d-9d14-100320582734.png)
